### ISSUE TITLE
i think the guy on reddit was right

### ISSUE THREAD

**Issue by naor173** [finndev/PokeBuddy#444]
they are flagged all the bots i think we better stop now until something safer comes out i think my both accounts flagged and going to get banned.. and they are planning to do a perm ban.. the message i got on pokeadvisor is really scary guys you should check your account as well go on pokeadvisor.com tell me if u have the same message that your blocked

**Comment by WilliamcaFox** [comment]
PokeAdvisor detects false cheaters based upon catch rate. It's not a legitimate resource or interpretation of what will happen. Pokeadvisor is not Niantic. Relax.

**Comment by naor173** [comment]
you sure?? because it says the source of the messege is niantic blocked you doesnt correct? it happens in all my accounts

**Comment by WilliamcaFox** [comment]
I am positive. This does not mean you account is blocked from the game. This means PokeAdvisor cannot refresh your data until Niantic improves Third Party Support.

> Unfortunately, it appears Niantic has blocked PokeAdvisor from accessing your data. You will not be able to refresh your character until this is resolved.

**Comment by naor173** [comment]
yup that is the message i get.. so if i stop bot for a while i play regular it might be removing by himself? and the reason it caused. is the excellent throws we get every second?

**Comment by WilliamcaFox** [comment]
This error is not caused by the bot. You turning the bot off won't fix their website.

**Comment by naor173** [comment]
but if i play normal for a while they see my % getting lower like it should be and i wont be flagged.. doesnt it make sense?

**Comment by naor173** [comment]
i still think something happend because i used a bot only last night with 1 account and i got this message on all my accounts i got 3.. they made a move is that possible?

**Comment by WilliamcaFox** [comment]
What I am saying is, the website  PokeAdvisor is BLOCKED completely. There is nothing you can do.

**Comment by naor173** [comment]
you sure it doesnt connect to niantic? and there is another website i can check pokemon %?

**Comment by creeperkilla720** [comment]
CALM down man nothing will happen what WilliamcaFox is saying is true dont worry but if youre still gonna do something about it just uncheck the box to catch pokemon and grind pokestops

**Comment by dodolock** [comment]
pokeadvisor=trash  period

**Comment by pokemonloverx333** [comment]
they check the stats anyway once they get this far , and if they see u catch around 100 pokemon in 1 hour or evolve 20 in 2 minutes they will just ban u cause if u click on evolve at your app on your mobile it takes about 15-20 seconds to be finished , while bots and api evolve 7 at same time , if they start banning , were all gonna be banned , thats why i made fun accounts botting up and playing regulary on my main , which is 21 btw now with normal non bot playing , just by going outside...
